code/game/objects/items/stunbaton.dm

```diff
@@ -12,7 +12,9 @@
 	attack_verb = list("beaten")
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 50, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)
 
-	var/stunforce = 140
+	var/stunforce = 40		//CITADEL EDIT							-- How much stamina damage is applied on hit.

```

Comment:
remember to adjust stunprod stunforce so that they don't become super ultra bullshit